packages/rum-core/src/domain/action/actionCollection.spec.ts

```diff
@@ -117,4 +118,24 @@ describe('actionCollection', () => {
     })
     expect(rawRumEvents[0].domainContext).toEqual({})
   })
+  it('should discard negative action loading time', () => {
```

Comment:
🥜 nit: shall we specify here we are discarding the field (do not set it) rather than the action when it's negative?

Reply:
yes great idea 👍